File: services/dbservice/src/main/java/com/torin/dbService/dto/HandlerDto.java
package com.torin.dbService.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class HandlerDto {
    private Long id;

    private Long apiId;

    private String hash;

    private String phone;

    private String directoryForUserPhoto;

    private String directoryForMedia;

    private String category;

    private Integer countGroup;

    private String nameHandler;
}

File: services/telegramservice/DTOs/DTO/HandlerDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class HandlerDTO
    {
        public long Id { get; set; }

        public long? ApiId { get; set; }

        public string? Hash { get; set; }

        public string? Phone { get; set; }

        public string? DirectoryForUserPhoto { get; set; }

        public string? DirectoryForMedia { get; set; }

        public string? Category { get; set; }

        public int? CountGroup { get; set; }

        public string? NameHandler { get; set; }
        public string? Status { get; set; }
    }
}